gold-zip-input.html

```diff
@@ -126,21 +126,23 @@
     _computeValue: function(value) {
       var start = this.$.input.selectionStart;
       var previousCharADash = this.value ? this.value.charAt(start - 1) == '-' : false;
-
-      // Remove any already-applied formatting.
-      value = value.replace(/-/g, '');
-
-      // Add a dash after the 5th character
-      if (value.length > 5) {
-        value = value.substr(0,5) + '-' + value.substr(5)
-      }
-      this.updateValueAndPreserveCaret(value.trim());
-
-      // If the character right before the selection is a newly inserted
-      // dash, we need to advance the selection to maintain the caret position.
-      if (!previousCharADash && this.value.charAt(start - 1) == '-') {
-        this.$.input.selectionStart = start + 1;
-        this.$.input.selectionEnd = start + 1;
+      
+      if (value) {
```

Comment:
This error checking could be called at the start of the function and simply be:
`if (!value) return;`